src/tequila/grouping/binary_rep.py

```diff
@@ -230,23 +232,41 @@ def anti_commutativity_matrix(self):
         gram = np.block([[np.zeros((n,n)), np.eye(n)], [np.eye(n), np.zeros((n,n))]])
         return matrix @ gram @ matrix.T % 2
 
-    def commuting_groups(self, method='rlf'):
+    def commuting_groups(self, method='rlf', condition=None, overlap_help=None):
         """
         Return the partitioning of the hamiltonian into commuting groups.
         List of BinaryHamiltonian's
         """
```

Comment:
Because it's not so obvious what the parameters are doing, some explanation following the format below is appreciated https://github.com/schoi839/tequila/blob/915605c2001c3f0839898944ad5d131e3436e0dc/src/tequila/circuit/circuit.py#L242-L263